virtManager/createvm.py

```diff
@@ -858,6 +859,9 @@ def _populate_machine(self):
 
         defmachine = None
         prios = []
+        if self._capsinfo.arch in ["sw_64"]:
+            defmachine = "core3"
```

Comment:
i think this should be done in `virtinst.Guest.get_recommended_machine()`, in the internal `_qemu_machine()` function of it: see the existing cases

Reply:
Thanks for the kind reminder, I've fixed it